requirements-freeze.txt

```diff
@@ -0,0 +1,32 @@
+bagit==1.7.0
```

Comment:
Definitely in favor of pip freeze inside container, but here, probably not? Our couple dependencies are stable, and we're not really in control of the envrionment where the cwl runs, in any case.

I could be wrong... but if you do want to keep it, then make sure it gets installed on travis, and update the readme: Otherwise it's just deceptive. (And that additional bookkeeping is why I don't think it's worth it.)

Reply:
Ok.